nss-day-cohort-48/the-worst-site-ever-fandango
# worked on the loading wheel

Worked on making the webpage even more HORRIBLE by adding a loading wheel and hiding the page content.


## Steps to test

1. git checkout ldu-wse3
2. check to make sure loading wheel operates correctly
3. check to make sure content at bottom is hidden until you scroll down to it
